Copyedit Correlation ID plugin documentation #3202 (#3258)
Kong/docs.konghq.com: app/_hub/kong-inc/correlation-id/index.md

ip-port-pid-connection-connection_requests-timestamp
```

This correlation id contains more practical implications for each request.

The following is a detailed description of the field

form parameter      | description
---                 | ---

> `ip` | an address of the server which accepts a request

Reviewer: ```suggestion
`ip` | an address of the server that accepts a request
```

> `port` | port of the server which accepts a request

Reviewer: ```suggestion
`port` | port of the server that accepts a request
```

`pid` | pid of the nginx worker process
`connection` | connection serial number
`connection_requests` | current number of requests made through a connection
`timestamp` | a floating-point number for the elapsed time in seconds (including milliseconds as the decimal part) from the epoch for the current timestamp from the nginx cached time

## FAQ

### Can I see my correlation ids in my Kong logs?

> The correlation id does not show up in the Nginx access or error logs. However, you can use this plugin along with one of the Logging plugins, or store this id on your backend-side.

Reviewer: ```suggestion
The correlation id doesn't show up in the Nginx access or error logs. However, you can use this plugin along with one of the Logging plugins, or store the id on your backend.
```
